### ISSUE TITLE
Travis pipeline not testing prod install

### ISSUE THREAD

**Issue by jamesbhobbs** [bbc/simorgh#1473]
**Describe the bug**
The travis pipeline is installing dev dependencies and therefore e2e, a11y and lighthouse tests aren't actually testing the production setup (there could be and have in the past been issues of missing production dependencies).

**To Reproduce**
This is readily visible in the install script: `npm ci`

**Expected behavior**
Test production install

**Screenshots**
n/a

**Desktop (please complete the following information):**
n/a

**Smartphone (please complete the following information):**
n/a

**Additional context**
n/a

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)
